app/views/shared/_user_sidebar_nav.html.erb

```diff
@@ -4,7 +4,7 @@
       <h>hi <%= current_user.first_name.downcase %></p>
     </li>
     <li class="divider"></li>
-    <li><%= link_to "dashboard", user_dashboard_path(current_user) %></li>
+    <li><%= link_to "dashboard", user_dashboard_path %></li>
     <li><%= link_to "order history", user_orders_path(current_user) %></li>
```

Comment:
looks like you don't need current_user for either user_orders_path or user_listings_path